File: app/src/main/java/com/devx/raju/utils/ShareUtils.java
package com.devx.raju.utils;

import android.app.Activity;
import androidx.core.app.ShareCompat;

public class ShareUtils {

    public static void shareUrl(Activity activity,
                                String url) {
        ShareCompat.IntentBuilder
                .from(activity)
                .setType("text/plain")
                .setChooserTitle("Share URL")
                .setText(url)
                .startChooser();
    }
}

File: app/src/main/java/com/devx/raju/utils/ShareUtils.kt
package com.devx.raju.utils

import android.app.Activity
import androidx.core.app.ShareCompat

object ShareUtils {
    fun shareUrl(activity: Activity?,
                 url: String?) {
        if (activity != null) {
            ShareCompat.IntentBuilder
                .from(activity)
                .setType("text/plain")
                .setChooserTitle("Share URL")
                .setText(url)
                .startChooser()
        }
    }
}